content/en/docs/ops/configuration/traffic-management/dns-proxy/index.md

```diff
@@ -135,10 +136,95 @@ $ kubectl exec deploy/sleep -- curl -sS -v auto.internal
 
 As you can see, the request is sent to an automatically allocated address, `240.240.0.1`. These addresses will be picked from the `240.240.0.0/16` reserved IP address range to avoid conflicting with real services.
 
+## External TCP services without VIPs
+
+By default, Istio has a limitation when routing external TCP traffic because it is unable to distinguish between multiple TCP services on the same port. This limitation is particularly apparent when using third party databases such as AWS Relational Database Service or any database setup with geographical redundancy. Similar, but different external TCP services, cannot be handled separately by default. For the sidecar to distinguish traffic between two different TCP services that are outside of the mesh, the services must be on different ports or they need to have globally unique VIPs.
+
+For example, if you have two external database services, `mysql-instance1` and `mysql-instance2`, and you create service entries for both, client sidecars will still have a single listener on `0.0.0.0:{port}` that looks up the IP address of only `mysql-instance1`, from public DNS servers, and forwards traffic to it. It cannot route traffic to `mysql-instance2` because it has no way of distinguishing whether traffic arriving at `0.0.0.0:{port}` is bound for `mysql-instance1` or `mysql-instance2`.
+
+The following example shows how DNS proxying can be used to solve this problem.
+A virtual IP address will be assigned to every service entry so that client sidecars can clearly distinguish traffic bound for each external TCP service.
+
+1.  Update the Istio configuration specified in the [Getting Started](#getting-started) section to also configure `discoverySelectors` that restrict the mesh to namespaces with `istio-injection` enabled. This will let us use any other namespaces in the cluster to run TCP services outside of the mesh.
```

Comment:
Not sure why discoverySelectors is required? Generally its best to keep examples as minimal as possible or it adds confusion about why it was set.

In this case, I  don't think its needed? Because we have auto-alloc we don't have issues with port conflicts.

Reply:
If discoverySelectors is not present, cannot simulate the problem of external service, because tcp-echo service which is within the cluster is getting discovered by default.

Reply:
I see. ExportTo or Sidecar may be a easier way to simulate it. And maybe we can mention its just for testing purposes, so we don't have to use an external site for testing, and not needed in real world?